src/components/shared/ButtonWithPanel.js

```diff
@@ -85,6 +86,7 @@ class ButtonWithPanel extends PureComponent {
               'buttonWithPanelButton',
               `${className}Button`
             )}
+            disabled={!!disabled}
```

Comment:
I checked that when it's `false` the attribute is not rendered at all in HTML. (it shows up in the test snapshots though).

I'm pretty sure this is the same for  `undefined` but I still prefered to force it into a boolean.